flybywiresim/aircraft
# fix(nd): fix active leg navaid color

Fixes #8691

## Summary of Changes

Previously, when the active leg termination was a tuned navaid, the symbol appeared in white. This is incorrect, tuned navaids should appear in cyan. This PR fixes it.

## Screenshots (if necessary)


![image](https://github.com/flybywiresim/aircraft/assets/22713769/38bf68ec-1d2b-4ac3-8f6f-c741d24b2e25)


## References

see issue #8691 

## Additional context

There are more errors in the ND symbology. For example, symbols from the EFIS overlay should be behind flightplan symbols. I don't have the capacity to fix these issues at the moment, so I will restrict the PR to this small fix.


Discord username (if different from GitHub):

## Testing instructions

- Load up on the runway, tune navaids on the RAD NAV page, make sure they appear in cyan.
- Make tuned navaids the active leg by using a DIR TO for example, check that the navaid is still cyan.
- Check different combinations of EFIS filters, active waypoints and tuned navaids.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
